source/database/mssql/scripts/dbscripts/PSP_PIMS_S52_00/Build/075_DML_PIMS_ROLE_CLAIM.sql

```diff
@@ -58,11 +58,11 @@ select @rolePimsR = CLAIM_ID FROM PIMS_CLAIM where NAME = 'ROLE_PIMS_R';
 DECLARE @adminUsers BIGINT;
 DECLARE @adminProjects BIGINT;
 DECLARE @adminProperties BIGINT;
-DECLARE @systemAdministrator BIGINT;
```

Comment:
This is an unrelated error in these scripts, there is a system-administrator claim and a System Administrator role. Thankfully this would not have caused adverse changes as both values have the same id (1).